src/assets/main.css

```
@@ -22,6 +22,11 @@ body.p-overflow-hidden {
   overscroll-behavior: contain !important;
 }
 
+.grecaptcha-badge {
+  bottom: 75px !important;
+  z-index: 999;
```

z-index muito alto. O footer tem 10.
```suggestion
  z-index: 11;
```